tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -351,11 +351,16 @@ def test_sharing(self, cluster, friend_account_logged_in_docker_cluster_pk_ssh):
         resource_class_name = cluster.config().get("resource_type").capitalize()
         config = cluster.config()
 
-        expected_creds = (
-            "ssh-sky-key"
-            if isinstance(cluster._creds, SSHSecret)
-            else f'{config["name"]}-ssh-secret'
-        )
+        default_keypair = rh.globals.configs.get("default_keypair")
+        if isinstance(cluster._creds, SSHSecret):
+            # If default keypair is specified, creds will be using that name
+            expected_creds = (
+                f"ssh-{os.path.basename(default_keypair)}"
```

Comment:
also it stores the secret name not path, so should be able to just use `expected_creds = default_keypair` here